### PR TITLE
Added documentation for tac.md

### PR DESCRIPTION
**Pull request by jskollin**
# Advanced HW 9 submission

Closes #120

Name: Julia Kollin
Uniqname: jskollin

### CODE REVIEW

File: tac.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+If no filename is provided, or the filename is inputted as "-", 'tac' reads input from the command line and prints it in reverse.
+
+### Additional Options/Examples
+
+~~~ bash
+$ tac [OPTION] [FILE]
+~~~
+
+By default, line separators are newlines and are placed after each line of output.  Use '-b' to place line separators before each line of output instead.
```
Comment (mterwill) [inline tac.md]: Please surround all command line arguments or excerpts in \`backticks\` so they appear `like this`.

File: tac.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+-before, -b
```
Comment (mterwill) [inline tac.md]: `--before` is a longopt

File: tac.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+This is because in the above example, the line separator is placed after the string, whereas using the -b option puts the line separator before the string. 
+
+--regex, -r
+
+This flag tells 'tac' to interpret the line separator as a regular expression.  This is commonly used with the --seperator flag.
+
+### Applications
+
+Error logs or any documents written in chronological order have the most recent information at the bottom of the file. 'tac' can provide easy access to that information without having to scroll through the entire file.
```
Comment (mterwill) [inline tac.md]: Nice use case!!

**Comment by themoh** [comment]
@mterwill does this satisfy everything you asked for?

**Comment by mterwill** [comment]
@themoh Still waiting on the above changes.

**Comment by themoh** [comment]
@jskollin We'll be closing all pull requests at midnight tonight, so please have this done by then.

**Comment by jskollin** [comment]
I made those edits.  Please let me know if there is anything else you need me to do. 

Thank you :)

**Comment by themoh** [comment]
Please move tac.md to _commands/tools

**Comment by the-alex** [comment]
I think you moved it to `_commands/tac.md`. I think @themoh is asking for it to be in `_commands/tools/tac.md`

**Comment by jskollin** [comment]
Do I have to move it in my virtual box terminal and push it? Or is there a way for me to do it on GitHub?

**Comment by the-alex** [comment]
I think you have to push it yourself. I'm not sure how to move files from within github. I know you can make small edits to the files themselves in the web interface.

**Comment by the-alex** [comment]
Done. 
7faa54b9e9d52dadcc74e0ab49bdf216825ce660

**Comment by jskollin** [comment]
THANK YOU :)

**Pull request merged**